### DISCUSSION TITLE
Failure to create remote configstore during create-remote-secret

### DISCUSSION THREAD

**Discussion by ravikgvs** [istio/istio#50021] [General]
![image](https://user-images.githubusercontent.com/119452228/204668716-1d5ab255-c3d5-4bc4-b740-8b34024b7328.png)

**Comment by hzxuzhonghu** [comment]
This is showing eror timeout listing CRDs, how many CRDs do you have? What if you increase the timeout here `pilot/pkg/config/kube/crdclient/client.go`
```
func knownCRDs(crdClient apiextensionsclient.Interface) (map[string]struct{}, error) {
	var res *crd.CustomResourceDefinitionList
	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

**Comment by howardjohn** [comment]
could also be complete lack of connectivity (firewall, etc)
